### PR TITLE
Adding Hope EDI dataset

### PR DESCRIPTION
**Pull request by jamespaultg**

### CODE REVIEW

File: datasets/hope_edi/hope_edi.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        ),
+        datasets.BuilderConfig(
+            name="tamil", version=VERSION, description="This part of my dataset covers Tamil dataset"
+        ),
+        datasets.BuilderConfig(
+            name="malayalam", version=VERSION, description="This part of my dataset covers Tamil dataset"
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "english"
```
Comment (lhoestq) [inline datasets/hope_edi/hope_edi.py:81]:
```suggestion
```

Not sure we want to have english as the default config over the other languages

File: datasets/hope_edi/hope_edi.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+        with open(filepath, encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(
+                csv_file, quotechar='"', delimiter="\t", quoting=csv.QUOTE_NONE, skipinitialspace=False
+            )
+
+            for id_, row in enumerate(csv_reader):
+
+                # train, validation
+                # While generating dummy data, the split skips the last tab in the last line, resulting in 2 values instead of 3
```
Comment (lhoestq) [inline datasets/hope_edi/hope_edi.py:162]: can we remove this check and only keep `text, label = row` then ?

**Review by lhoestq — COMMENTED** [review]
Nice thanks ! That's a nice dataset

I left a few comments

**Comment by jamespaultg** [comment]
@lhoestq Have addressed your comments. Please review. Thanks.

**Review by lhoestq — APPROVED** [review]
Thanks !

**Pull request merged**